docs/components/query.md

```diff
@@ -336,6 +336,7 @@ Flags:
                                  if no max_source_resolution param is specified.
       --query.partial-response   Enable partial response for queries if no
                                  partial_response param is specified.
+				 (--no-query.partial-response for disabling)
```

Comment:
Missing a space I think (at least on github view).

Reply:
Also maybe add it to the cli help output ?

Reply:
nah, everything is aligned if you go to file view mode.

Reply:
added to cli help output

Reply:
yeah, added this goddamn whitespace